camelot/admin/application_admin.py

```diff
@@ -377,10 +372,17 @@ def get_main_menu(self) -> MenuItem:
     def get_name(self):
         return 'application'
 
-    def get_version(self):
-        """:return: string representing version of the application, by default this
-                    is the class attribute verion"""
-        return self.version
+    def get_build_tag(self):
```

Comment:
die hele bazaar mag hier wat mij betreft gewoon weg hoor

waar je de info nodig hebt haal je hem uit de root backend, die toch een singleton is ...